charts/seed-bootstrap/charts/elastic-kibana-curator/templates/elasticsearch/service.yaml

```diff
@@ -8,9 +8,14 @@ metadata:
     role: logging
 spec:
   ports:
-  - port: {{ .Values.global.elasticsearchPorts.db }}
+  - name: db
+    port: {{ .Values.global.elasticsearchPorts.db }}
     protocol: TCP
     targetPort: http
+  - name: metrics
+    port: 9108
```

Comment:
`9114` is the [default port](https://github.com/prometheus/prometheus/wiki/Default-port-allocations) for the elastic search exporter. Any reason for using port `9108` instead?

Reply:
I took the port from the helm chart [here](https://github.com/helm/charts/tree/master/stable/elasticsearch-exporter)

Reply:
Hmm strange I guess since it doesn't change anything then this is fine